ddp-workspace/projects/ddp-atcp/src/app/app.module.ts

```diff
@@ -0,0 +1,196 @@
+}
+
+declare let DDP_ENV: any;
+
+export const tkCfg = new ToolkitConfigurationService();
+tkCfg.studyGuid = DDP_ENV.studyGuid;
+tkCfg.aboutYouGuid = 'ABOUTYOU';
+tkCfg.aboutChildGuid = 'ABOUTCHILD';
+tkCfg.consentGuid = 'CONSENT';
+tkCfg.consentAssentGuid = 'CONSENT_ASSENT';
+tkCfg.parentalConsentGuid = 'PARENTAL_CONSENT';
+tkCfg.releaseGuid = 'RELEASE_SELF';
+tkCfg.releaseMinorGuid = 'RELEASE_MINOR';
+tkCfg.dashboardGuid = 'DASHBOARD';
+tkCfg.lovedOneGuid = 'LOVEDONE';
+tkCfg.lovedOneThankYouGuid = 'THANK_YOU';
+tkCfg.aboutYouUrl = 'about-you';
+tkCfg.aboutChildUrl = 'about-your-child';
+tkCfg.lovedOneUrl = 'loved-one';
+tkCfg.consentUrl = 'consent';
+tkCfg.consentAssentUrl = 'consent-assent';
+tkCfg.parentalConsentUrl = 'parental-consent';
+tkCfg.releaseMinorUrl = 'release-minor-survey';
+tkCfg.releaseUrl = 'release-survey';
+tkCfg.dashboardUrl = 'dashboard';
+tkCfg.doneUrl = 'proxy-thank-you';
+tkCfg.activityUrl = 'activity';
+tkCfg.errorUrl = 'error';
+tkCfg.stayInformedUrl = 'stay-informed';
+tkCfg.lovedOneThankYouUrl = 'loved-one-thank-you';
+tkCfg.infoEmail = 'bistline@broadinstitute.org';
+
+// to configure feed, go to: https://lightwidget.com/widget-info/814feee04df55de38ec37791efea075e
+// need Instagram credentials for @osteosarcomaproject
+tkCfg.lightswitchInstagramWidgetId = '814feee04df55de38ec37791efea075e';
+tkCfg.countMeInUrl = 'https://joincountmein.org/';
+
+export let config = new ConfigurationService();
+config.backendUrl = DDP_ENV.basePepperUrl;
+config.auth0Domain = DDP_ENV.auth0Domain;
+config.auth0ClientId = DDP_ENV.auth0ClientId;
+config.studyGuid = DDP_ENV.studyGuid;
+config.logLevel = LogLevel.Info;
+config.baseUrl = location.origin + base;
+config.auth0SilentRenewUrl = DDP_ENV.auth0SilentRenewUrl;
+config.loginLandingUrl = DDP_ENV.loginLandingUrl;
+config.auth0CodeRedirect = location.origin + base + 'auth';
+config.localRegistrationUrl = config.backendUrl + '/pepper/v1/register';
+config.doLocalRegistration = DDP_ENV.doLocalRegistration;
+config.mapsApiKey = DDP_ENV.mapsApiKey;
+config.auth0Audience = DDP_ENV.auth0Audience;
+config.projectGAToken = DDP_ENV.projectGAToken;
+
+export function translateFactory(translate: TranslateService, injector: Injector) {
+  return () => new Promise<any>((resolve: any) => {
+    const locationInitialized = injector.get(LOCATION_INITIALIZED, Promise.resolve(null));
+    locationInitialized.then(() => {
+      const locale = 'en';
+      translate.setDefaultLang(locale);
+      translate.use(locale).subscribe(() => {
+        console.log(`Successfully initialized '${locale}' language as default.`);
+      }, err => {
+        console.error(`Problem with '${locale}' language initialization.`);
+      }, () => {
+        resolve(null);
+      });
+    });
+  });
+}
+
+export class MyHammerConfig extends HammerGestureConfig {
+  public buildHammer(element: HTMLElement): Hammer {
+    const hammer = new Hammer(element, {
+      touchAction: 'pan-y'
+    });
+    return hammer;
+  }
+}
```

Comment:
It is also Osteo thing, if you don't need to handle gestures on mobile devices, please, get rid of it.